package.json

```diff
@@ -13,12 +13,10 @@
     "tar"
   ],
   "devDependencies": {
-    "node-gyp": "3.x",
     "tap": "7.x"
   },
   "scripts": {
-    "test": "tap --reporter tap tests/api_tests.js",
-    "install": "node-gyp rebuild"
```

Comment:
no, node-gyp isn't in your PATH. npm has a version internally that it uses, but its not accessible to you. by specing it as a dev dep, it gets installed as a dep, and ./node_modules/.bin/ is added to the PATH by npm when running scripts so that dependency's scripts are in the path. Its ok, this is just a minor cleanup, if it causes cascading changes in the build tooling, its not worth it (at least now).